NugetVisualizer/Core/ConfigurationHelper.cs

```diff
@@ -17,7 +17,7 @@ private IConfigurationSection GetSection(string sectionName)
         }
 
         public bool UseSqlLite => true;
-        public string Dbpath => GetSection("Dbpath").Exists() ? GetSection("Dbpath").Value : "nugetvisualizer.db";
+        public string Dbpath => GetSection("Dbpath").Exists() ? GetSection("Dbpath").Value : "..\\nugetvisualizer.db";
```

Comment:
this is not platform agnostic, will fail on linux

Reply:
will take care of this :)